ekf/tests/test_runner/config_file_handler.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * Phoenix-Pilot
+ *
+ * Part of EKF test runner
```

Comment:
```suggestion
 * EKF test runner config preparation routines
```
Please amend header file accordingly.